### ISSUE TITLE
Question in Tutorial docs

### ISSUE THREAD

**Issue by vamsi-98** [apple/pkl#177]
In the [example](https://pkl-lang.org/main/current/language-tutorial/01_basic_config.html#entries) for entries I see that object is used as key. But when I evaluate the example it throws the below error 

<img width="862" alt="image" src="https://github.com/apple/pkl/assets/42208075/cc3958f8-c583-42f4-a844-a245807fa3b8">

Is the code in the docs correct or am I missing something?

**Comment by stackoverflow** [comment]
Indeed, this example doesn't render properly to `json` or `yaml`, just to `pcf`. Probably a good idea to change it to something that's `json` renderable.

**Comment by vamsi-98** [comment]
I was thinking it can be changed to something like this, not sure if it exactly highlights the purpose of the example for entries.

```
pigeonShelter {
  ["bird"] {
    name = "Common wood pigeon"
    diet = "Seeds"
    taxonomy {
      species = "Columba palumbus"
    }
  }
  ["address"] = "355 Bird St."
}
count = "value"
birdCount {
  ["value"] = 42
```
